jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1039,19 +1040,23 @@ public void testGetTaskById() {
     	assertEquals("Write a Document", userTask.getName());
 
     	NodeInstanceDesc nodeInstanceDesc = runtimeDataService.getNodeInstanceForWorkItem(userTask.getWorkItemId());
-		assertNull(userTask.getSlaCompliance());
-		assertNull(userTask.getSlaDueDate());
+    	assertNull(userTask.getSlaCompliance());
+    	assertNull(userTask.getSlaDueDate());
 
-		userTask = runtimeDataService.getTaskById(taskId,true);
-		assertNotNull(userTask);
-
-		assertEquals(nodeInstanceDesc.getSlaCompliance(), userTask.getSlaCompliance());
-		assertEquals(nodeInstanceDesc.getSlaDueDate(), userTask.getSlaDueDate());
+    	userTask = runtimeDataService.getTaskById(taskId, true);
+    	assertNotNull(userTask);
+    	assertEquals(nodeInstanceDesc.getSlaCompliance(), userTask.getSlaCompliance());
+    	assertEquals(nodeInstanceDesc.getSlaDueDate(), userTask.getSlaDueDate());
+    	
+    	assertNotNull(userTask.getCorrelationKey());
```

Comment:
Not enought check also the value of correlation key. as a rule of thumb the correlation key by default is the process instance id to string but you can start a process with a custom correlation key. (do that)